go/vt/vtgate/planbuilder/throttler.go

```diff
@@ -48,3 +48,27 @@ func buildShowThrottledAppsPlan(query string, vschema plancontext.VSchema) (*pla
 		Query:             query,
 	}), nil
 }
+
+func buildShowThrottlerStatusPlan(query string, vschema plancontext.VSchema) (*planResult, error) {
+	dest, ks, tabletType, err := vschema.TargetDestination("")
+	if err != nil {
+		return nil, err
+	}
+	if ks == nil {
+		return nil, vterrors.NewErrorf(vtrpcpb.Code_FAILED_PRECONDITION, vterrors.NoDB, "No database selected: use keyspace<:shard><@type> or keyspace<[range]><@type> (<> are optional)")
+	}
+
+	if tabletType != topodatapb.TabletType_PRIMARY {
+		return nil, vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "SHOW VITESS_THROTTLER STATUS works only on primary tablet")
```

Comment:
How would you adjust it? The way I see it it is consistent with responses to other types of queries.

Reply:
```
return nil, vterrors.NewErrorf(vtrpcpb.Code_FAILED_PRECONDITION, vterrors.NoDB, "No database selected: use keyspace<:shard> or keyspace<[range]> (<> are optional)")
```

Since they cannot specify the type (at least non-primary).

Reply:
done